mapss/static-media/packages/arches/arches/app/templates/base.htm

```diff
@@ -36,12 +36,12 @@
     <meta name="description" content="">
     <meta name="author" content="">
 
-    <link rel="shortcut icon" href="{% static 'img/favicon.png' %}" />
+    <link rel="shortcut icon" href="{% webpack_static 'img/favicon.png' %}" />
 
     {% block css %}
-        <link href="{% static 'css/arches.css' %}" rel="stylesheet">
-        <link href="{% static 'css/package.css' %}" rel="stylesheet">
-        <link href="{% static 'css/project.css' %}" rel="stylesheet">
+        <link href="{% webpack_static 'css/arches.css' %}" rel="stylesheet">
+        <link href="{% webpack_static 'css/package.css' %}" rel="stylesheet">
+        <link href="{% webpack_static 'css/project.css' %}" rel="stylesheet">

```

Comment:
There could be other places that need this change, but this was enough to get project CSS loaded correctly in the custom widget I'm working on.